conf/vertebrates/mlss_conf.xml

```diff
@@ -261,7 +261,7 @@
         <genome name="cyclopterus_lumpus" assembly="fCycLum1.pri"/>
         <genome name="nothobranchius_furzeri" assembly="Nfu_20140520"/>
         <genome name="oncorhynchus_kisutch" assembly="Okis_V2"/>
-        <genome name="cyprinus_carpio"/>
+        <genome name="cyprinus_carpio_carpio" assembly="Cypcar_WagV4.0"/>
```

Comment:
The inclusion of the assembly name here has implications for how the assembly is used.

If this genome is intended to be a reference only until a future assembly update replaces it, then the assembly name should be included in the genome element.

If, on the other hand, this assembly is to be preserved in the EPO MSA indefinitely, the assembly attribute should be omitted.

To put this in context, all of the other AQUA-FAANG species with the exception of _Sparus aurata_ have been configured without the assembly name.

Reply:
so we need to remove the assembly name for all aqua-faang species then.

Reply:
Assembly name removed.